evadb/readers/pdf_reader.py

```diff
@@ -34,10 +34,12 @@ def _read(self) -> Iterator[Dict]:
         doc = fitz.open(self.file_url)
 
         # PAGE ID, PARAGRAPH ID, STRING
+        # Maintain a global paragraph number per PDF
+        global_paragraph_no = 0
```

Comment:
I added this so that the calculated id for the create index method stays unique. Otherwise, it is still not unique because multiple paragraphs at different pages can have the same paragraph id. @gaurav274